data/data/azure/vnet/nsg.tf

```diff
@@ -0,0 +1,69 @@
+resource "azurerm_network_security_group" "master" {
+  name                = "${var.cluster_id}-master-nsg"
+  location            = "${var.region}"
+  resource_group_name = "${var.resource_group_name}"
+
+  security_rule {
+    name                       = "AllowSSH"
+    priority                   = 100
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "22"
+    destination_port_range     = "22"
+    source_address_prefix      = "*"
```

Comment:
https://github.com/openshift/installer/blob/master/data/data/aws/vpc/sg-master.tf#L44

Reply:
the NSG is assigned to the subnet. no need to specify the CIDR